airflow/cli/cli_parser.py

```diff
@@ -1362,7 +1362,8 @@ class GroupCommand(NamedTuple):
     ),
     ActionCommand(
         name='downgrade',
-        help="Downgrade the schema of the metadata database",
+        help="Downgrade the schema of the metadata database.  Must provide either `--revision` "
```

Comment:
The `help` field should contain one sentence. Additional information can be added in `description` field. We use `help` field to display cheat-sheet, so we have to be careful that this command is still readable and only contains the key information.

Reply:
ok thanks will adjust it

Reply:
@mik-laj where is the cheat sheet?  i tried adding validation for `help` command to enforce max length but found other long help text examples.  e.g. `Clean up Kubernetes pods` command.  I tried searching, but found only [this page](https://airflow.apache.org/docs/apache-airflow/stable/cli-and-env-variables-ref.html?highlight=clean%20up%20kubernetes%20pods) in the documentation and the length is not an issue there.

thanks

Reply:
We have a separate command that displays a cheatsheet. So far, only two commands have a description that requires a wrap when the terminal width is 120.
<img width="844" alt="Screenshot 2022-03-06 at 19 02 33" src="https://user-images.githubusercontent.com/12058428/156935855-1a591014-4af1-42f5-8114-e3528ac7b709.png">
<img width="841" alt="Screenshot 2022-03-06 at 19 02 44" src="https://user-images.githubusercontent.com/12058428/156935856-7a5c7636-2cea-48ba-a323-7f932fc0be6f.png">

Reply:
Ah perfect thank you I'll take care of this